app/js/views/app_view.js

```diff
@@ -1,15 +1,42 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todo_item_collection',
+  'models/todo_item'
+], function ($, Backbone, indexTemplate, TodoItemCollection, TodoItem) {
   return Backbone.View.extend({
     el: this.$('#todo-app'),
 
+    initialize: function(){
+      this.todoItemCollection = new TodoItemCollection();
+    },
+
+    events:{
+      "click #add-todo-button" : "addTodo",
+      "keydown #todo-title-textbox" : "onKeyDownEventHandler"
```

Comment:
Please be consistent in naming event handlers. Suggestion `on{elementName}{event}`. Ex: `onAddTodoButtonClick`